tests/testthat/test-check_tbl_values_required.R

```diff
@@ -277,7 +277,10 @@ test_that("(#123) check_tbl_values_required works with all optional output types
   )
   # Ensure that req_vals check is the only one that fails
   expect_snapshot(
-    check_for_errors(validate_submission(hub_path, file_path)),
+    check_for_errors(validate_submission(
+      hub_path, file_path,
+      skip_submit_window_check = TRUE
```

Comment:
Just fixing an oversight in a previously added test that was now returning a submission window check error